### ISSUE TITLE
Whitespace missing while updating imports

### ISSUE THREAD

**Issue by ChillarAnand** [alecthomas/importmagic#28]
`import pandas as pd` is getting converted to `import pandasas pd`

Steps to reproduce:

```python
In [10]: python_source
Out[10]: 'import pandas as pd\n\n\nfrom django.core.management.base import BaseCommand\n\n\n\npd.read_csv()\nx = BaseCommad\n'

In [11]: unresolved, unreferenced = scope.find_unresolved_and_unreferenced_symbols()

In [12]: importmagic.update_imports(python_source, index, unresolved, unreferenced)
Out[12]: 'import pandasas pd\n\n\npd.read_csv()\nx = BaseCommad\n'
```

**Comment by alecthomas** [comment]
I believe this is fixed in master. I will do a new release shortly.

**Comment by alecthomas** [comment]
I just released 0.1.5 which has a lot of fixes. I'm going to close this under the assumption that the issues are fixed. Reopen it if not.

**Comment by ChillarAnand** [comment]
Fixed in 0.1.5. Thank you :-)
